block-layer/codecs/dag-json.md

```diff
@@ -20,9 +20,14 @@ producing the same data end up with matching block hashes.
 
 All kinds of the IPLD Data Model except Bytes and Link are supported natively by JSON.
 
+#### Numbers
+
+Numbers are a special case. JSON only has a single number type, though many dynamically typed languages (e.g. Python, Ruby, PHP) distinguish between integers and floats when parsing JSON. A number consisting only of digits is parsed as integer, if it contains a dot, it's parsed as a float. In DAG-JSON the same method is used to represent integers and floats.
```

Comment:
for the sake of consistency, perhaps wrap this paragraph like the others?

Reply:
I wanted to keep the diff small.

Reply:
Sorry, you meant the opposite. There are already paragraphs that aren't wrapped. So it's mixed already anyway.